src/ui/components/shared/Forms/Radio.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function Radio({ id, name, checked, onChange }: React.HTMLProps<HTMLInputElement>) {
```

Comment:
Not sure about this suggestion, so feel free to ignore, but maybe we could do `overrides` and `{...overrides}` on line 7?

Reply:
👍